File: 2562-find-the-array-concatenation-value/2562-find-the-array-concatenation-value.java
class Solution {
    public long findTheArrayConcVal(int[] nums) {
        String str="";
        long sum=0;
        int i=0;
        int j=nums.length-1;
        
        while(i<=j)
        {
            if(i==j)
            {
                str=String.valueOf(nums[i]);
                sum+=Long.parseLong(str);
            }
            else
            {
                str=String.valueOf(nums[i])+String.valueOf(nums[j]);
                sum+=Long.parseLong(str);
            }
            i++;
            j--;
        }
        return sum;
    }
}

File: 2562-find-the-array-concatenation-value/2562-find-the-array-concatenation-value.cpp
class Solution {
public:
    long long findTheArrayConcVal(vector<int>& nums) {
        long long result=0;
        string str="";
        int i=0;
        int j=nums.size()-1;
        
        while(i<=j)
        {
            if(i==j)
            {
                str+=to_string(nums[i]);
            }
            else
            {
                str+=to_string(nums[i])+to_string(nums[j]);
            }
            i++;
            j--;
            result+=stoi(str);
            str="";
        }
        return result;
    }
};